reference/5.1/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -528,17 +474,16 @@ methods of the `System.Management.ManagementDateTimeConverter` class.
 </Type>
 ```
 
-`<ScriptProperty>`: Defines a property whose value is the output of a
-script.
+`<ScriptProperty>`: Defines a property whose value is the output of a script.
 
-The `<ScriptProperty>` tag must have a pair of `<Name>` tags that specify
-the name of the new property and a pair of `<GetScriptBlock>` tags
-that enclose the script block that returns the property value.
+The `<ScriptProperty>` tag must have a pair of `<Name>` tags that specify the
+name of the new property and a pair of `<GetScriptBlock>` tags that enclose the
+script block that returns the property value.
 
-For example, the `VersionInfo` property of files (`System.IO.FileInfo`
-objects) is a script property that results from using the `FullName`
-property of the `GetVersionInfo` static method of
-`System.Diagnostics.FileVersionInfo` objects.
+For example, the **VersionInfo** property of files (`System.IO.FileInfo`
```

Comment:
I don't like how this reads (similar to directories).  Maybe make files singular and pull "objects" out of the parenthesis?  Or leave it alone...

Reply:
Fixed